### DISCUSSION TITLE
MultiSelect: Show item categories when selecting items

### DISCUSSION THREAD

**Discussion by fernandes77** [mantinedev/mantine#3220] [Feature requests]
### Inspiration: Slack
<img width="741" alt="2022-12-21_15-51" src="https://user-images.githubusercontent.com/48812482/208981773-be1d23e2-d9e3-4476-a33d-b68c0b5519d5.png">

We want to display selected items of the same category in the same box and after the category name.

E.g.: On Slack, I want to see messages from #tech and #general. So when I select them, they appear as **`in: #tech, #general`**.

### Possible solution

The type SelectItem can have another property named `category`. For the example above, value of category would be `in`.

When an item that has this `category` gets selected, it gets put inside the little box with `in:` before it. Same goes for other items of the same category.

**Comment by rtivital** [comment]
You can change how values are rendered by following this guide – https://mantine.dev/core/multi-select/#custom-label-component
